eng/tools/mcp/azure-sdk-java-mcp/src/update-changelog-md.ts

```diff
@@ -0,0 +1,71 @@
+import path from "path";
+import { findAzureSdkRoot, findModuleDirectory } from "./utils/sdk-directory.js";
+import { getJavaSdkChangelogJson } from "./java-sdk-changelog.js";
+import { CallToolResult } from "@modelcontextprotocol/sdk/types";
+import * as fs from "fs";
+
+export async function updateChangelogMd(jarPath: string, groupId: string, artifactId: string): Promise<CallToolResult> {
```

Comment:
This seems like another place where we already have common code in the azure-sdk-tools repo to handle changelog parsing and updating.

Reply:
It compares two jars, so it is java specific. But I think we can have all languages support update changelogs tool to make the user experience consistent. 
Also as Weidong mentioned, some part of this tool can be done by engsys agentm e.g. update the latest session in CHANGELOG.